yt/yt/core/https/server.cpp

```diff
@@ -150,7 +156,9 @@ IServerPtr CreateServer(
 
 IServerPtr CreateServer(const TServerConfigPtr& config, const IPollerPtr& poller)
 {
-    return CreateServer(config, poller, poller);
+    YT_LOG_WARNING_IF(config->Credentials->UpdatePeriod,
+        "TLS certificate update runs in poller invoker");
```

Comment:
Why is it WARNING?

Reply:
https://github.com/ytsaurus/ytsaurus/pull/384#discussion_r1510335021
Currently this function is used only in tests and does not enable updates.
Do you prefer assert?